stable/redis/templates/deployment.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: 1
```

Comment:
well let me ask a question, cause maybe I do not understand. @prydonius does this setup support more than one replica? I am thinking that it does not, but I could be wrong. That means if I can that set this to two, or attempt to scale this I have standalone masters running.

So a couple of options.
1. Document in readme thay this is standalone redis, not HA
2. Document to not scale this here, or maybe in the readme

Just thinking of a n00b user installing this and going ... Wait a second I scaled this but it is not replicating.

But great chart, not going to argue about super detailed docs. I just know and love this app a ton.